src/sentry/uptime/grouptype.py

```diff
@@ -27,8 +28,7 @@ class UptimeDomainCheckFailure(GroupType):
             "properties": {
                 "mode": {
                     "type": ["integer"],
-                    # TODO: Enable this when we can move this grouptype out of this file
-                    # "enum": [mode.value for mode in ProjectUptimeSubscriptionMode],
+                    "enum": [mode.value for mode in ProjectUptimeSubscriptionMode],
```

Comment:
This is going to break when I import it into the models. We need to move this mode thing into `uptime.types`

Reply:
Why do you need to import the grouptype into models?

Reply:
As an example

```py
@cache_func_for_models([(Detector, get_org_from_detector)])
def get_active_auto_monitor_count_for_org(organization: Organization) -> int:
    return Detector.objects.filter(
        type=UptimeDomainCheckFailure.slug,
        project__organization=organization,
        config__mode__in=[
            ProjectUptimeSubscriptionMode.AUTO_DETECTED_ONBOARDING,
            ProjectUptimeSubscriptionMode.AUTO_DETECTED_ACTIVE,
        ],
    ).count()
```